src/app/services/backend.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Http } from '@angular/http';
```

Comment:
This file is where I can feel the technical debt / shoddy patchwork code accumulating. It needs a proper scrubbing, but for this iteration, it works and moves us forward.